doc/docs/FAQ.md

```diff
@@ -316,11 +316,15 @@ No. Meep does not support non-orthogonal grids with spatially varying resolution
 
 In principle, this corresponds to the limit as the frequency goes to zero or the wavelength goes to infinity.  However, a time-domain simulation is rather inefficient for such [electrostatic](https://en.wikipedia.org/wiki/Electrostatics) (or magnetostatic) calculation; this includes [lumped circuit models](https://en.wikipedia.org/wiki/Lumped_element_model) such as resistance, voltage, capacitance, etc. In this regime, you are usually much better off directly solving e.g. [Poisson's equation](https://en.wikipedia.org/wiki/Poisson%27s_equation#Electrostatics) to obtain the fields from a given charge distribution. There are many available Poisson solvers based on [finite](https://en.wikipedia.org/wiki/Finite_element_method) or [boundary](https://en.wikipedia.org/wiki/Boundary_element_method) element methods.  In Meep, probably the best you can do is to use a source with a very low frequency and a gradual turn-on specified by the `width` parameter of [`ContinuousSrc`](Python_User_Interface.md#continuoussource).
 
-### Why does the continuous-wave (CW) source produce a finite bandwidth response?
+### Why doesn't the continuous-wave (CW) source produce a single-frequency response?
 
-The [ContinuousSource](Python_User_Interface.md#continuoussource) does not produce an exact single frequency response due to its finite turn-on time (which is described by a hyperbolic tangent function). If you Fourier transform the response, the finite turn-on time will produce other frequency components. After the turn-on, the continuous source is described by a sinusoid of the given frequency.
+The [ContinuousSource](Python_User_Interface.md#continuoussource) does not produce an exact single frequency response due to its finite turn-on time which is described by a hyperbolic tangent function. If you Fourier transform the response, the finite turn-on time will produce additional frequency components. After the turn-on, the CW source is described by a sinusoid of the given frequency.
```

Comment:
I would say that asymptotically the resulting fields *are* the single frequency response.  It's just that if you Fourier transform the whole simulation you will see a finite bandwidth due to the turn-on.